data/json/items/armor.json

```diff
@@ -5697,6 +5715,24 @@
     "material_thickness": 1,
     "flags": [ "WATER_FRIENDLY", "OUTER", "SPLINT" ]
   },
+  {
+    "id": "arm_xlsplint",
+    "type": "ARMOR",
+    "name": "arm splint",
+    "description": "A tool to help set bones and hold them in place.",
+    "weight": 300,
+    "volume": 4,
+    "price": 20000,
+    "material": [ "wood", "cotton" ],
+    "symbol": "[",
+    "color": "white",
+    "covers": [ "ARM_EITHER" ],
```

Comment:
This entry is basically a copy of the first, and duplicates the arm_splint, when I believe you were intending this to be a leg XL splint.

Additionally, same as comment above; need to update the coverage, encumbrance, weight, etc.